tests/unit/resizable-panels.spec.ts

```diff
@@ -34,4 +34,59 @@ describe('Resizable Panels', () => {
     expect(wrapper.exists()).toBeTruthy();
     expect(wrapperRightPanelChild.exists()).toBeTruthy();
   });
+
+  describe('left panel', () => {
+    it('should have the default ratio apply to its width', () => {
+      const wrapper = shallowMount(ResizablePanels);
+      const panels: WrapperArray<any> = wrapper.findAll('.resizable-panels__panel');
+
+      expect(panels.at(0).element.style.width).toEqual('40%');
+    });
+  });
+
+  describe('right panel', () => {
+    it('should have the default ratio apply to its width', () => {
+      const wrapper = shallowMount(ResizablePanels);
+      const panels: WrapperArray<any> = wrapper.findAll('.resizable-panels__panel');
+
+      expect(panels.at(1).element.style.width).toEqual('60%');
+    });
+  });
+
+  describe('when resizing', () => {
+    it('should know that is being resized', () => {
+      const wrapper = shallowMount(ResizablePanels, {
+        attachToDocument: true,
+      });
+      const resizerWrapper = wrapper.find('.resizable-panels__resizer');
+
+      resizerWrapper.trigger('mousedown');
+      wrapper.trigger('mousemove', { movementX: 10 });
+      expect(wrapper.vm.$data.isResized).toBeTruthy();
```

Comment:
Isn't `rapper.vm.isResized` sufficient?

I'm also not sure we should test for internal data properties.
What is the effect of this data on the rendered component?
If there is one, I think that's what we should test for. Otherwise, the test can be dropped.

Reply:
That was TypeScritpt compiler that enforce that. Indeed, the `vm` object don't know about internal vars.
But you're right, that property isn't used so I remove it from the component.